packages/jest-editor-support/src/Settings.js

```diff
@@ -73,16 +70,18 @@ module.exports = class Settings extends EventEmitter {
       // See https://github.com/facebook/jest/issues/2343 for moving this into
       // the config object
       if (string.includes('jest version =')) {
-        const version = string.split('jest version =')
+        const version = string
+          .split('jest version =')
           .pop()
-          .split(EOL)[0]
-          .trim();
+          .split(EOL)
+          [0].trim();
         this.jestVersionMajor = parseInt(version, 10);
       }
 
       // Pull out the data for the config
       if (string.includes('config =')) {
-        const jsonString = string.split('config =')
+        const jsonString = string
+          .split('config =')
           .pop()
           .split('No tests found')[0];
```

Comment:
If there's a MemberExpression last but no CallExpression afterwards, then we group it. This way you don't have `.required` inside of a React PropType be on its own line and look weird.